packages/EasyApiToken/src/Bridge/Symfony/Resources/config/services.php

```diff
@@ -21,5 +21,5 @@
 
     $services
         ->set(ApiTokenDecoderInterface::class)
-        ->factory([ref(ApiTokenDecoderFactoryInterface::class), 'buildDefault']);
+        ->factory([service(ApiTokenDecoderFactoryInterface::class), 'buildDefault']);
```

Comment:
As we discussed above we can't use this function.
And we already have an opened [PR](https://github.com/eonx-com/easy-monorepo/pull/825) that replaces `ref` with `service`.

Reply:
done